crosspoint_reader/driver.py

```
@@ -149,23 +149,36 @@ def config_widget(self):
-    def books(self, oncard=None, end_session=True):
-        if oncard is not None:
-            return []
-        entries = self._http_get_json('/api/files', params={'path': '/'})
-        books = []
-        fetch_metadata = PREFS['fetch_metadata']
+    def _list_files_recursive(self, path='/'):
+        """Return a flat list of (lpath, size) for all EPUB files on device."""
+        results = []
+        try:
+            entries = self._http_get_json('/api/files', params={'path': path})
+        except Exception as exc:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)